docs/index.md

```diff
@@ -1,8 +1,8 @@
 Welcome to Stellio documentation!
 =================================
 
-[![fiware_core](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/core.svg)](https://www.fiware.org/developers/catalogue/)
-[![fiware_stackoverflow](https://nexus.lab.fiware.org/repository/raw/public/badges/stackoverflow/fiware.svg)](https://stackoverflow.com/questions/tagged/fiware/)
+[![fiware_core](style/core-badge.svg)](https://www.fiware.org/developers/catalogue/)
+[![fiware_stackoverflow](style/stackoverflow-badge.svg)](https://stackoverflow.com/questions/tagged/fiware/)
```

Comment:
the FIWARE badges are also broken in the main README.md. Can you fix them too in this PR?